pkg/tss/capabilities.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package tss provides TPM 1.2/2.0 core functionality and
```

Comment:
the package comment must be in _one_ file only, not in all of them.

Reply:
done